chain/chain/src/types.rs

```diff
@@ -156,21 +156,13 @@ impl BlockEconomicsConfig {
     /// been overwritten at specific protocol versions. Chains with a genesis
     /// version higher than those changes are not overwritten and will instead
     /// respect the value defined in genesis.
-    pub fn min_gas_price(&self, protocol_version: ProtocolVersion) -> Balance {
-        if self.genesis_protocol_version < MIN_PROTOCOL_VERSION_NEP_92 {
-            if protocol_version >= MIN_PROTOCOL_VERSION_NEP_92_FIX {
-                MIN_GAS_PRICE_NEP_92_FIX
-            } else if protocol_version >= MIN_PROTOCOL_VERSION_NEP_92 {
-                MIN_GAS_PRICE_NEP_92
-            } else {
-                self.genesis_min_gas_price
-            }
-        } else if self.genesis_protocol_version < MIN_PROTOCOL_VERSION_NEP_92_FIX {
-            if protocol_version >= MIN_PROTOCOL_VERSION_NEP_92_FIX {
-                MIN_GAS_PRICE_NEP_92_FIX
-            } else {
-                MIN_GAS_PRICE_NEP_92
-            }
+    pub fn min_gas_price(&self, _protocol_version: ProtocolVersion) -> Balance {
+        if self.genesis_protocol_version < _UNUSED_MIN_PROTOCOL_VERSION_NEP_92 {
+            // Protocol version is always >= MIN_PROTOCOL_VERSION_NEP_92_FIX
+            MIN_GAS_PRICE_NEP_92_FIX
+        } else if self.genesis_protocol_version < _UNUSED_MIN_PROTOCOL_VERSION_NEP_92_FIX {
```

Comment:
Remove condition `self.genesis_protocol_version < _UNUSED_MIN_PROTOCOL_VERSION_NEP_92`.

Rename `_UNUSED_MIN_PROTOCOL_VERSION_NEP_92_FIX` to `MIN_PROTOCOL_VERSION_NEP_92_FIX`.

Remove all comments about `Protocol version is always >= MIN_PROTOCOL_VERSION_NEP_92_FIX`